### PR TITLE
Minor style fixes

### PR DESCRIPTION
**Pull request by Kahn**
You beat me to fixing the extra fence :) While I was here I fixed a few other minor markdown nits.

### CODE REVIEW

**Comment by drowe67** [comment]
@Kahn lol yes I just happened to doing a test install yesterday so tweaked a few things. Thanks for your PR, I will take a look...

File: README.md [hunk 1]
```diff
@@ -1,26 +1,39 @@
...
 + Listens for FreeDV signals, then transmits a reply.
 + Supports FreeDV 1600/700C/700D modes.
 + Places the received signal files on a web server.
 + Requires a Linux machine with a sound card and RS232-PTT (or Hamlib CAT) interface to your radio.
 + Just one sound card is required.
 + Can run on machines as small as a Raspberry Pi.
 
-When a "trigger" string is detected in the rx FreeDV text message (e.g. "hello beacon", or the beacon callsign), the beacon will transmit a signal report back to you.
+When a "trigger" string is detected in the rx FreeDV text message (e.g. "hello beacon", or
+the beacon callsign), the beacon will transmit a signal report back to you.
```
Comment (drowe67) [inline README.md:14]: @Kahn can you pls explain the changes here?  Are you adding newlines?

**Comment by drowe67** [comment]
@Kahn looks good :+1: 

Here I am testing the effect of ```sh```:

```
cd codec2/build_linux
make
```

```sh
cd codec2/build_linux
make
```

OK, so syntax highlighting - cool.

File: README.md (hunk 1 above)
Comment (Kahn) [reply README.md:14]: Yup, might be overkill here but in general try and keep markdown line lengths between 80 - 100 characters for readability in an editor. The rendered output remains a single paragraph.

It's ultimately optional though, just a force of habit for me :)

**Comment by Kahn** [comment]
Yep! Full list of languages supported seems to be https://github.com/github/linguist/blob/master/lib/linguist/languages.yml

File: README.md (hunk 1 above)
Comment (drowe67) [reply README.md:14]: Yes I've adopted (are in the process of adopting) the convention of no line breaks, so we better stick with that in this file too ....

Comment (drowe67) [reply README.md:14]: @Kahn could you pls :point_up:

**Pull request closed without merging**